.gitignore

```diff
@@ -1,4 +1,46 @@
-node_modules
-.idea
-.cache
```

Comment:
this is still needed

Reply:
I replaced the entire .gitignore with the one that Angular CLI generated, and it still includes `/node_modules` and `/.idea`. Not sure if we need `/.cache`.

Reply:
leave it just in case. Having already run `parcel build`, my directory is full of files